### PR TITLE
Triggers from "help ben", "it is raining" and "oh yeah"

### PR DESCRIPTION
**Pull request by danielthepope**
I heard on Gitter earlier "We need your help Ben" and "It is raining", but JSOxBot failed to respond :(
I considered allowing "Its raining" (without the apostrophe) but I decided poor grammar doesn't deserve an enlightening response.
I thought I'd take off the exclamation mark from "oh yeah" while I was at it too.
Also I'm okay with tabs and I'm okay with spaces, but I'm not okay with both being used at the same time for indentation ;)

### CODE REVIEW

File: scripts/benroll.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    msg.send("[This is relevant to the conversation](http://www.bensmindpalace.co.uk/)");
   });
-  robot.hear(/it's raining/i, function(msg){
-  	msg.send("It's raining and @benfoxall is still wearing shorts.");
+  robot.hear(/it('| i)s raining/i, function(msg){
+    msg.send("It's raining and @benfoxall is still wearing shorts.");
   });
-  robot.hear(/Oh yeah!/i, function(msg){
-  	msg.send("![](https://files.gitter.im/jsoxford/jsoxford.github.com/JSbn/Ben.gif)");
+  robot.hear(/Oh yeah/i, function(msg){
```
Comment (AverageMarcus) [inline scripts/benroll.js:13]: Might be worth changing this to `/Oh yeah(^|!|.)/` so we don't pick up mid-sentence. E.g. "Oh yeah thats what it was"

Comment (danielthepope) [inline scripts/benroll.js:13]: Could be done with `/Oh yeah($|!|\.)/i`, but I thought you'd want it to pick up on a sentence like that.

Comment (AverageMarcus) [inline scripts/benroll.js:13]: Ugh! Yeah, that's what I meant. It's been a long day.

Comment (danielthepope) [inline scripts/benroll.js:13]: No worries, it's keeping me on my toes ;)

**Comment by danielthepope** [comment]
This has conflicts and I've deleted my fork. Never mind!

**Pull request closed without merging**